tests/TestCase/Http/ServerRequestTest.php

```diff
@@ -731,6 +734,10 @@ public function testReferer()
         $result = $request->referer(true);
         $this->assertSame('/some/path', $result);
 
+        $request->env('HTTP_REFERER', Configure::read('App.fullBaseUrl') . '///cakephp.org/');
```

Comment:
Whats the value here?
I would expect a test like `//my-domain/path` as test to test the above code change.

Reply:
Sorry if I am wrong, but the problem is at line 578 in ServerRequest.php:
```php
$ref = substr($ref, strlen($base));
```
This removes `App.fullBaseUrl` (e.g. `http://localhost`) and `ServerRequest::$webroot` (e.g. `/`) from the `HTTP_REFERER`. So I tested `http://localhost` + `/` + `//cakephp.org/` here.